includes/set-partuuid

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# shamelessly copied from https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=196778
```

Comment:
Are we allowed to do this from a licensing standpoint? Just checking.

Reply:
Dunno. It's a post offering the code for download, with no license to apply (at least I have not seen any).
So crediting the origin is _more_ than the author asked for in terms of licensing, isn't it ?

Reply:
Sure, it is now. I don't think it matters too much I was more of just curious.